metadata-ingestion/src/datahub/cli/delete_cli.py

```diff
@@ -213,6 +213,17 @@ def references(urn: str, dry_run: bool, force: bool) -> None:
         logger.info(f"Deleted {references_count} references to {urn}")
 
 
+@delete.command()
+@click.option("--urn", required=True, type=str, help="the urn of the entity")
+def undo_by_filter(urn: str) -> None:
```

Comment:
there's no filters supported here - so the `undo_by_filter` method name is a bit strange

Reply:
it allows us to add them in future